lib/main.rb

```diff
@@ -0,0 +1,24 @@
+#./log/games.log
+
+class Main
+   def initialize(path)
+      @path = path
+   end
+
+   def fopen
+      if(File.exists?(@path))
+         @file = File.open(@path)
+         if(@file != nil)
+            @line = @file.readlines.map(&:chomp)
```

Comment:
Não tem necessidade de declarar variáveis de instância durante a execução do código, costuma ser uma má prática quando não são declaradas no initializer.
Sugiro a leitura:
https://www.campuscode.com.br/conteudos/variaveis-e-self-em-ruby

```suggestion
            line = @file.readlines.map(&:chomp)
```